terms/demographics/raceDetail.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.raceDetail-0.0.1",
+  "type": "string",
+  "description": "Race detail"
```

Comment:
I don't understand what 'race detail' is. What is this term expected to be used for?

Reply:
Agreed, I don't know what this means either

Reply:
Is this being used in AD or PEC

Reply:
It's in the PEC individual human template but it doesn't appear to have any values according to the PEC clinical table.

Reply:
Could this definition be updated to be more descriptive of the intent? "Detailed race information"? That doesn't really add all that much clarity, though...

Reply:
@amapeters @kelshmo Has this key been used in another consortium where we could get more information on its use?  Since it doesn't appear that anyone has used it yet in PEC, can we just drop it?

Reply:
I think we can drop it